app/controllers/users.rb

```diff
@@ -0,0 +1,30 @@
+module Chitter
+
+  module Routes
+
+    class Users < Base
+
+      get '/users/new' do
+        @user = User.new
+        erb :'users/new'
+      end
+
+      post '/users' do
+        @user = User.new(email: params[:email],
```

Comment:
Here you could pass the params hash to User.new, rather than extracting out each values. Of course, you then risk users submitting random form data which will cause your app to crash. So you then might have a helper method within each controller that would filter the params hash for only those attributes permitted for the creation of the associated resource (in this case, User). This is something Rails offers: http://edgeapi.rubyonrails.org/classes/ActionController/StrongParameters.html